src/Samples/PX.Objects.HackathonDemo/PX.Objects.HackathonDemo/Graph/GoTo Test/APInvoiceEntry.cs

```diff
@@ -125,8 +125,6 @@ public IEnumerable items()
 				return rows;
 			else
 				return rows;
-
-			return rows;
```

Comment:
This unreachable code was specifically left there as far as I remember to show that PX1015 is not shown for unreachable code

Reply:
Returned it back.